Source/Coop/Components/ActionPacketHandlerComponent.cs

```diff
@@ -302,10 +302,10 @@ bool ProcessWorldPacket(ref Dictionary<string, object> packet)
                 //    ReplicateRaidTimer(packet);
                 //    result = true;
                 //    break;
-                case "TimeAndWeather":
-                    ReplicateTimeAndWeather(packet);
-                    result = true;
-                    break;
+                //case "TimeAndWeather":
+                //    ReplicateTimeAndWeather(packet);
+                //    result = true;
+                //    break;
```

Comment:
I think this can be removed, you removed the function that is in this commented out code as well :stuck_out_tongue: